coderd/notifications/notifications_test.go

```diff
@@ -774,7 +776,9 @@ func TestNotificationTemplatesCanRender(t *testing.T) {
 			payload: types.MessagePayload{
 				UserName: "Bobby",
 				Labels: map[string]string{
-					"deleted_account_name": "bobby",
+					"deleted_account_name":      "bobby",
+					"deleted_account_user_name": "william tables",
```

Comment:
Why lower case here (cmp. above)? Feels like there's a bug when looking at golden files.

Reply:
fixed in subsequent PR